### ISSUE TITLE
Allow stdin redirection with exec command

### ISSUE THREAD

**Issue by theoriginalgri** [tutumcloud/cli#71]
It would be great to be able to redirect a file to stdin of the exec command. For example when importing a sql dump to a testing container: ```tutum exec db-1 sudo -u postgres psql dbname < database.dump```

Unfortunately this fails currently with the following error message:
```
$ tutum exec db-1 less < test
Traceback (most recent call last):
  File "/usr/local/Cellar/tutum/0.16.0.1/libexec/bin/tutum", line 9, in <module>
    load_entry_point('tutum==0.16.0.1', 'console_scripts', 'tutum')()
  File "/usr/local/Cellar/tutum/0.16.0.1/libexec/lib/python2.7/site-packages/tutumcli/tutum_cli.py", line 321, in main
    dispatch_cmds(args)
  File "/usr/local/Cellar/tutum/0.16.0.1/libexec/lib/python2.7/site-packages/tutumcli/tutum_cli.py", line 123, in dispatch_cmds
    commands.container_exec(args.identifier, args.command)
  File "/usr/local/Cellar/tutum/0.16.0.1/libexec/lib/python2.7/site-packages/tutumcli/commands.py", line 613, in container_exec
    invoke_shell(url)
  File "/usr/local/Cellar/tutum/0.16.0.1/libexec/lib/python2.7/site-packages/tutumcli/commands.py", line 519, in invoke_shell
    oldtty = termios.tcgetattr(sys.stdin)
termios.error: (25, 'Inappropriate ioctl for device')
```

**Comment by neilellis** [comment]
+1

**Comment by smazurov** [comment]
I'm having the same issue, but for a different reason.  I have a jenkins server and when I try to execute a tutum exec it dies with the same error.  I'm assuming this would get fixed if you guys work on this issue, so I am adding a :+1: as this issue is forcing us to do some workarounds.

**Comment by tifayuki** [comment]
fixed in version 0.16.21
